lib/client/ui/plugins/camera.js

```diff
@@ -58,16 +58,14 @@ module.exports = {
             capture.setAttribute("controls",true);
             capture.style.display = "inline";
             capture.style.height = (screen.availHeight - 50) + "px";
-            capture.src = window.webkitURL.createObjectURL(upload.files[0]);
+            capture.src = (window.URL || window.webkitURL).createObjectURL(upload.files[0]);
```

Comment:
Yeah, browsers that required the webkit prefix seemed to switch to plain URL a couple of years ago (though not sure when exactly Opera switched), but I figured there was no harm leaving it in there for now.